src/preset.ts

```diff
@@ -807,6 +807,9 @@ export async function expandConfigurePreset(folder: string, name: string, worksp
     if (!preset) {
         return null;
     }
+    if (preset.installDir) {
```

Comment:
I think this needs to go in the if statement on line 852. We first need to expand any macros in the `installDir`, then we can normalize the path

Reply:
ok this time it should be in the right place haha. `resolvePath` also normalizes the path: 
```
export function resolvePath(inpath: string, base: string) {
    const abspath = path.isAbsolute(inpath) ? inpath : path.join(base, inpath);
    // Even absolute paths need to be normalized since they could contain rogue .. and .
    return lightNormalizePath(abspath);
}
```